{"name": "test_02_fio[sudo fio -filename=\\u6587\\u4ef6 -direct=1 -iodepth 1 -thread -rw=read -ioengine=psync -bs=512b -size=1G -group_reporting -name=mytest]", "status": "broken", "statusDetails": {"message": "TypeError: not all arguments converted during string formatting", "trace": "self = <Case.test_fio.TestIOzone object at 0x000002488820BE10>\ncmd = 'sudo fio -filename=文件 -direct=1 -iodepth 1 -thread -rw=read -ioengine=psync -bs=512b -size=1G -group_reporting -name=mytest'\n\n    @pytest.mark.description(\"fio硬盘性能测试: \"\n                             \"http://qa-zentao.mthreads.com:9000/testcase-view-8388-7.html?tid=xen4k1po\")\n    @pytest.mark.parametrize('cmd', [i[0] for i in data_fio])\n    def test_02_fio(self, cmd):\n        fio_rs = fp.send(cmd)\n>       final_rs = fp.assert_fio(fio_rs)\n\ntest_fio.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\Utils\\Base_init.py:102: in assert_fio\n    logging.info(IOPS,dict_list['IOPS'],dict_list['IOPS'])\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:2195: in info\n    root.info(msg, *args, **kwargs)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1519: in info\n    self._log(INFO, msg, args, **kwargs)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1664: in _log\n    self.handle(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1680: in handle\n    self.callHandlers(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1736: in callHandlers\n    hdlr.handle(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1026: in handle\n    self.emit(record)\nC:\\Users\\yi.mei-ext\\AppData\\Roaming\\Python\\Python313\\site-packages\\_pytest\\logging.py:386: in emit\n    super().emit(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1158: in emit\n    self.handleError(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1150: in emit\n    msg = self.format(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:998: in format\n    return fmt.format(record)\nC:\\Users\\yi.mei-ext\\AppData\\Roaming\\Python\\Python313\\site-packages\\_pytest\\logging.py:139: in format\n    return super().format(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:711: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <LogRecord: root, 20, C:\\Programs\\BSPTest\\Utils\\Base_init.py, 102, \"84.9\">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:400: TypeError"}, "attachments": [{"name": "log", "source": "8b2e5a33-c707-4a50-8d30-89d6ea70a5c9-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "cmd", "value": "'sudo fio -filename=文件 -direct=1 -iodepth 1 -thread -rw=read -ioengine=psync -bs=512b -size=1G -group_reporting -name=mytest'"}], "start": 1741170193891, "stop": 1741170256068, "uuid": "f2148170-27bb-418c-b625-dfd2a30d5b8a", "historyId": "9f895ae1a138433a8e6e8c2c0ba35b96", "testCaseId": "39baddf0d196c4f8caec87c0d2e6ef3d", "fullName": "Case.test_fio.TestIOzone#test_02_fio", "labels": [{"name": "tag", "value": "description('fio硬盘性能测试: http://qa-zentao.mthreads.com:9000/testcase-view-8388-7.html?tid=xen4k1po')"}, {"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_fio"}, {"name": "subSuite", "value": "TestIOzone"}, {"name": "host", "value": "Meiyi-EXT"}, {"name": "thread", "value": "39456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_fio"}]}